Sources/Keyboard/KeyboardLayout.swift

```diff
@@ -5,8 +5,11 @@ public enum KeyboardLayout {
     /// Traditional Piano layout with raised black keys over white keys
     case piano
     
-    /// All notes linearly right afater one another
+    /// All notes linearly right after one another
     case isomorphic
+    
+    /// Notes in 4ths layout with rows and columns
+    case guitar
```

Comment:
Actually, the guitar layout should have an associated value for rowCount.

Reply:
If you had an associated value for intervalBetweenRows rowCount, that would cover a lot of cases. I'd like to call them strings, since you're already calling it guitar layout, but of course a string is very different in Swift.